product-service/src/main/resources/application-dev.yml

```
@@ -70,4 +70,4 @@ management:
   endpoints:
     web:
       exposure:
-        include: health,prometheus
+        include: health,prometheus
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)